rpc/rpcserver/server.go

```diff
@@ -44,10 +44,10 @@ import (
 
 // Public API version constants
 const (
-	semverString = "2.2.0"
+	semverString = "2.2.1"
```

Comment:
This PR needs to be rebased over master.  We're already at 2.2.1 and this needs to be 2.2.2.